### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by Awokein**
Adjusted for clarity, and removed unnecessary hyphen.

Please replace this sentence with the details of your pull request; make sure you leave in and **tick** the declarations below.

**Declarations**:

- [ ✅] I declare that this contribution was created in whole or in part by me
- [✅ ] I declare that I have the right to submit this contribution under the terms of this repository's license and declarations.
- [✅ ] I understand and agree that this contribution and a record of it are public, maintained permanently, and may be redistributed infinitely under the terms of this repository's license.

In other words, you assert that you hold the copyright for your contribution and you agree that your contribution may be redistributed under [the terms of the license](https://github.com/evaera/Cmdr/blob/master/LICENSE)

### CODE REVIEW

**Comment by evaera** [comment]
This is spam. Please do not submit PRs like this.
